src/averageAges.js

```diff
@@ -14,12 +14,19 @@
  *
  * @return {number}
  */
+function avarageCalculator(ages, peopleCount) {
+  return ages.reduce((sum, age) => sum + age, 0) / peopleCount;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(man => (
+    century
+      ? man.sex === 'm' && Math.ceil(man.died / 100) === century
+      : man.sex === 'm'
```

Comment:
You are duplicating this expression `man.sex === 'm'` twice. Maybe, let's create the variable for it ?